src/native/fs/fs_napi.cpp

```diff
@@ -55,6 +56,16 @@ struct Entry
     uint8_t type;
 };
 
+
+struct KeyValue {
+    std::string key;
+    std::string value;
+};
+
+
+std::string USER_MD_PREFIX = "user.s3.";
```

Comment:
I think we better move it to config.js and send as arg to native calls.
We want this to be easily configurable. Certainly in the current stage.